smart-contracts/contracts/interfaces/IPublicLock.sol

```diff
@@ -233,6 +233,17 @@ contract IPublicLock is IERC721Enumerable {
     bytes calldata _data
   ) external payable;
 
+  /**
+   * @notice returns the minimum price paid for a purchase with these params.
```

Comment:
Just to clarify, this is the minimum price to allow for tips, correct?

Reply:
Yup this should be clarified after the refactor.